cvIgnacioCalahorro.html

```diff
@@ -45,16 +45,10 @@ <h2 class='about'>ABOUT ME:</h2>
         <div id="aboutmostrar" >
             <p class="texto-justificado">
                 I was granted an Erasmus scholarship to work as Intern at the University of Twente, Netherlands to
-                develop a GIS-Tool used in urban planning. I had been working as a Land Surveyor for EPSA International (LLOTOR Arabia) in an open air mine in Saudi Arabia property of Ma'aden Gold. Then I moved to Riyadh (KSA) where I was working for APPLUS+ under the managed of the largest consortium, FAST, as a surveyor throught the Line 5 of the Riyadh Metro Project.
+                develop a GIS-Tool used in urban planning. I had been working as a Land Surveyor for EPSA International (LLOTOR Arabia) in an open air mine in Saudi Arabia property of Ma'aden Gold. Afterwards, I moved to Riyadh (KSA) where I was working for APPLUS+ under the management of the largest consortium, FAST, as a surveyor through the Line 5 of the Riyadh Metro Project.
             </p>
             <p class="texto-justificado">
-                Later I was the Supervisor at Proyecto San Diego SL in Granada, Spain where I supervise a building
-                restoration and manage the rentals of the apartments. I was in Granada during the time I was expeting to
-                move to Alborg, Denmark to start the Msc Surveying and Mapping. While finished my studys there I got a
-                job
-                in Jaén as Asset Manager. Right after graduation I got the opportunity to work at Vexcel Spain where I
-                find
-                myself today
+                Later, I was the Supervisor at Proyecto San Diego SL in Granada, Spain where I supervised building restoration and managed the rentals of the apartments. Eventually, I started a master's programme in Surveying and Mapping in Aalborg, Denmark, and while finishing the studies, I started a job as Asset Manager in Jaén, Spain. Right after graduation I got the opportunity to work at Vexcel Spain where I find myself today.
```

Comment:
- The first sentence is in the past tense, but "supervise" and "manage" is the present tense
- The second sentence was overall messed up, completely reworked that. 
  - "I was in Granada during the time I was expecting to move to Aalborg" is just a very weird phrasing.
  - "MSc" (not "Msc" btw) is the degree you receive, but the programme is a master's programme (although to be fair it is commonly used interchangeably, but it's not correct).
  - "Jaén" -> "Jaén, Spain": I think it's better to be consistent. If you're specifying Aalborg, Denmark, Granada, Spain, etc then you should also specify Jaén, Spain.